frontend/src/features/layout/Sidebar/shell/Sidebar.scss

```
@@ -0,0 +1,144 @@
+
+	@media (min-width: 768px) {
+		z-index: +5;
+	}
+
+	position: fixed;
+	display: block;
+	top: var(--header-height);
+	left: 0;
+	bottom: 0;
+	width: var(--sidebar-width);
+	// height: calc(100vh - var(--header-height));
```

Stylelint (17.11.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)